R/pkg/R/functions.R

```diff
@@ -90,8 +90,11 @@ NULL
 #'
 #' String functions defined for \code{Column}.
 #'
-#' @param x Column to compute on. In \code{instr}, it is the substring to check. In \code{format_number},
-#'          it is the number of decimal place to format to.
+#' @param x Column to compute on except in the following methods:
+#'      \itemize{
+#'      \item \code{instr}: class "character", the substring to check.
```

Comment:
the form `class "character"` might not be commonly use around here.
how about like https://stat.ethz.ch/R-manual/R-devel/library/utils/html/read.table.html, just say
```
instr: character, the substring to check.
```